digital_land/expectations/log.py

```diff
@@ -0,0 +1,105 @@
+import csv
+import json
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+from pathlib import Path
+
+
+# TODO Review and connect to centralised Log classes do we need our own class for this
+class ExpectationLog:
```

Comment:
Can this inherit from the base Log class? (digital_land/log.py)

Reply:
I thikn yes in the. future but let's leave as separate for now and combine at a later date